src/reconcile/rec_write.c

```diff
@@ -1372,7 +1403,7 @@ __rec_txn_read(WT_SESSION_IMPL *session, WT_RECONCILE *r,
 		 */
 		if (vpack != NULL &&
 		    vpack->raw == WT_CELL_VALUE_OVFL_RM &&
-		    !__wt_txn_visible_all(session, min_txn))
+		    !__wt_txn_visible_all(session, min_txn, NULL))
```

Comment:
On review, I think we need to track the update with the oldest transaction ID, and check if it is `visible_all`.  As long as some update is stable, we don't need to copy the original.